tests/automation-framework/rancherclient/rancherclient.go

```diff
@@ -0,0 +1,79 @@
+package rancherclient
+
+import (
+	projectClient "github.com/rancher/rancher/tests/automation-framework/client/generated/project/v3"
+	"github.com/rancher/rancher/tests/automation-framework/clientbase"
+	"github.com/rancher/rancher/tests/automation-framework/config"
+	managementClient "github.com/rancher/rancher/tests/automation-framework/management"
+	"github.com/rancher/rancher/tests/automation-framework/testsession"
+	"k8s.io/client-go/rest"
+)
+
+type RancherClient struct {
+	Management    *managementClient.Client
+	Project       *projectClient.Client
+	RestConfig    *rest.Config
+	RancherConfig *config.RancherServerConfig
+}
+
+// NewClient is returns a larger client wrapping individual api clients.
+func NewClient(restConfig *rest.Config, testSession *testsession.TestSession) (*RancherClient, error) {
+	c := &RancherClient{
+		RestConfig: restConfig,
+	}
+
+	err := c.newManagementClient(testSession)
```

Comment:
it might be better to get rid of these 2 constructor methods and just create the clients in here.

Reply:
Sounds good I'll update that